From 4d09d105875aa631d864999124a441d0db961fff Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Fri, 17 Aug 2018 16:38:40 +0800
Subject: [PATCH 219/706] arm64: dts: ls1028a: Correct DMA-CH-MUX mapping
 numbers for SAI

This patch corrects DMA-Channel-MUX peripheral mapping numbers for SAI1
and SAI2.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit f3518edb7d9a90c2c10e667c9eaa1331289720ba)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index ce0b2ab9424b..dc50a9115852 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -544,8 +544,8 @@
 				 <&clockgen 4 3>, <&clockgen 4 3>;
 			clock-names = "bus", "mclk1", "mclk2", "mclk3";
 			dma-names = "tx", "rx";
-			dmas = <&edma0 1 47>,
-			       <&edma0 1 46>;
+			dmas = <&edma0 1 4>,
+			       <&edma0 1 3>;
 			status = "disabled";
 		};
 
@@ -558,8 +558,8 @@
 				 <&clockgen 4 3>, <&clockgen 4 3>;
 			clock-names = "bus", "mclk1", "mclk2", "mclk3";
 			dma-names = "tx", "rx";
-			dmas = <&edma0 1 45>,
-			       <&edma0 1 44>;
+			dmas = <&edma0 1 6>,
+			       <&edma0 1 5>;
 			status = "disabled";
 		};
 	};
-- 
2.17.1

